### ISSUE TITLE
By Tuesday morning: Commit changes made in #46 to code & add calculated fields

### ISSUE THREAD

**Issue by bnchdrff** [ESDet/esdportal_profile#48]
After #46 is done, dev can ensure bundle modifications are expressed in code (test deployment!) & add calculated fields as described in the notes.

**Comment by bnchdrff** [comment]
Additional tasks:

* Add link style to tables found in /my-teachers and /e3-log

**Comment by anderoy** [comment]
@bnchdrff #46 is all set to go. Please don't forget the link styles item in the comment above. Thanks!

P.S. I think I followed and respected your ideal work flow on this one. I think it worked well, no? Thanks for pushing me to document, it forced me to be more thoughtful and surprisingly efficient in my work.

**Comment by bnchdrff** [comment]
code is all in order, will push to prod tomorrow earlymorn!

**Comment by bnchdrff** [comment]
Did a local test deployment & verified all documented additions are a-ok.

Also fixed that e3-contact/%/teacher_report bug.

Deployed to live... lookin good!

**Comment by anderoy** [comment]
Looks good. Also the first time in a while that ALL 100% EVERY view is "In Code: although that will probably change quickly as I build the one-off editing views and exports that don't need to exist long- term.

Now we pause for a brief moment as I figure out what is next. Likely, we will be shifting away from the E3 side of the Portal and doing a few things for the Scorecard update. Will scope/plan it out in a new Google doc and create a new issue when the time comes.
